src/observer/utils.py

```diff
@@ -163,6 +162,20 @@ def to_dict(instance, descend_into_mn_fields=False):
             data[f.name] = f.value_from_object(instance)
     return data
 
+def safe_json_dumps(data, **kwargs):
+    "handles serialisation of certain objects. unserialisable objects become '[unserialisable]'"
+    def coerce(val):
+        "coerce a value to a serialisable value"
+        lu = {
+            datetime: ymdhms,
+            date: ymd,
+        }
+        if type(val) in lu:
+            return lu[type(val)](val)
+
+        #raise TypeError('Object of type %s with value of %s is not JSON serializable' % (type(obj), repr(obj)))
+        return "[unserialisable]"
```

Comment:
why not an explicit error? This string will propagate through reports (potentially in Nifi) as incorrect data